Project.toml

```diff
@@ -7,14 +7,20 @@ version = "0.1.0-DEV"
 AtomsBase = "a963bdd2-2df7-4f54-a1ee-49d51e6be12a"
 AtomsCalculators = "a3e0e189-c65a-42c1-833c-339540406eb1"
 Folds = "41a02a25-b8f0-4f67-bc48-60067656b558"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 
 [compat]
 AtomsBase = "0.3"
 AtomsCalculators = "0.1.2"
 Folds = "0.2.10"
-julia = "1.9"
 Unitful = "1"
+julia = "1.9, 1.10"
```

Comment:
Having "1.9" in compat includes "1.10", so just have "1.9" and thats it.

The same thing for LinearAlgebra and Printf